source/js/mozfest-event-handlers.js

```diff
@@ -0,0 +1,45 @@
+const bindHomeBannerHandlers = () => {
+  let homepage = document.querySelector("#view-mozfest-home");
+  if (!homepage) {
+    return;
+  }
+
+  let homeVideoControl = homepage.querySelector(".btn-video-control");
+  let homeVideo = homepage.querySelector("video.banner-video");
+
+  if (homeVideoControl && homeVideo) {
+    let showVideoControl = () => {
+      homeVideoControl.classList.remove(`invisible`);
+    };
+
+    homeVideoControl.addEventListener(`click`, () => {
+      if (!homeVideo.paused) {
+        homeVideo.pause();
+        homeVideoControl.innerText = `Resume`;
```

Comment:
I wonder if it would be possible to hide/show HTML elements instead of having copy in JS? We won’t be able to localize that otherwise

Reply:
Thanks @TheoChevalier . Yea I planned on doing that. I'm still waiting on designers to let me know what we wanna do to the button after users click on it.  Was just playing around to see what it would look like if button text changes 😄

Reply:
You rock 👊 Sorry for jumping the gun!